pkg/reconciler/openshift/tektonpipelinesascode/extension.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2021 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tektonpipelinesascode
+
+import (
+	"context"
+
+	"github.com/tektoncd/operator/pkg/reconciler/common"
+)
+
+func OpenShiftExtension(ctx context.Context) common.Extension {
+	return common.NoExtension(ctx)
+}
+
+//type openshiftExtension struct{}
+//
+//func (oe openshiftExtension) Transformers(comp v1alpha1.TektonComponent) []mf.Transformer {
+//	return nil
+//}
+//func (oe openshiftExtension) PreReconcile(context.Context, v1alpha1.TektonComponent) error {
+//	return nil
+//}
+//func (oe openshiftExtension) PostReconcile(ctx context.Context, comp v1alpha1.TektonComponent) error {
+//
+//	return nil
+//}
+//func (oe openshiftExtension) Finalize(context.Context, v1alpha1.TektonComponent) error {
+//	return nil
+//}
```

Comment:
shall we keep the `return nil` implementation for all these and return an instance of openshiftExtension instead of common.NoExtension. 

I think it makes it clear that where to add OpenShift specific changes in the future.